reconcile/utils/terrascript_aws_client.py

```diff
@@ -2451,6 +2455,49 @@ def populate_tf_resource_s3_cloudfront(self, resource, namespace_info):
             '${' + cf_oai_tf_resource.id + '}'
         tf_resources.append(Output(output_name_0_13, value=output_value))
 
+        # aws_s3_bucket_acl
+        values = common_values.get('distribution_config', {})
+        if 'logging_config' in values.keys():
+            values = {}
+            values['name'] = identifier
```

Comment:
missing `values["bucket"]`. The value should be taken from `logging_config.bucket`. Beware, the format is probably not the good one.